loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/LoadFlowProvider.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.loadflow;
+
+import com.powsybl.commons.Versionable;
+import com.powsybl.computation.ComputationManager;
+import com.powsybl.iidm.network.Network;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public interface LoadFlowProvider extends Versionable {
```

Comment:
workingStateId fixed.
- Normally yes because as it is an asynchronous call, another thread will be used, so the setWorkingVariant is not supposed to be done in the caller thread. But are we supposed in the thread taken from the pool to restore the initial working variant id?
- I would say no and encourage the use of default ForJoin.commonPool() as it is automatically sized according to number of core of the machine.

Reply:
OK.
Not related to this PR, but in this case maybe we should remove the `getExecutor` from `ComputationManager` interface ?
If it's not the executor to be used for computations, I think it's not in the right place.